addons/io/org.openhab.io.homekit/README.md

```diff
@@ -71,34 +65,34 @@ A full list of supported accessory types can be found in the table below.
  </tr>
  <tr>
   <td>&nbsp;</td>
-  <td>currentTemperature</td>
+  <td>CurrentTemperature</td>
   <td>Number</td>
-  <td>The current temperature, using the same unit rules as TemperatureSensor</td>
+  <td>The current temperature, same as above</td>
  </tr>
  <tr>
   <td>&nbsp;</td>
-  <td>heatingCoolingMode</td>
+  <td>homekit:HeatingCoolingMode</td>
```

Comment:
I will also add that the homekit binding has some properties that have to be set as config options, like 
```
org.openhab.homekit:useFahrenheitTemperature=true
org.openhab.homekit:thermostatCoolMode=CoolOn
org.openhab.homekit:thermostatHeatMode=HeatOn
org.openhab.homekit:thermostatAutoMode=Auto
org.openhab.homekit:thermostatOffMode=Off```

The temperature format is very important to other integrations, I would propose this be moved to a tag that can be set on the "Thermostat" .  The heating modes would also be nice to be able to set not through a specific binding.